automationcommon/middleware.py

```diff
@@ -5,6 +5,21 @@ class RequestUserMiddleware(object):
     """
     Middleware that simply set's the request.user to be used for the audit trail.
     """
+    def __init__(self, get_response):
+        self.get_response = get_response
+        # One-time configuration and initialization.
+
+    def __call__(self, request):
+        # Code to be executed for each request before
+        # the view (and later middleware) are called.
+
+        response = self.get_response(request)
```

Comment:
Shouldn't this be:
```python
response = self.get_response(self.process_request(request))
return self.process_response(request, response)
```
otherwise this middleware does diddly squit.